sdk/python/feast/infra/materialization/contrib/bytewax/bytewax_materialization_engine.py

```diff
@@ -273,7 +272,7 @@ def _print_pod_logs(self, job_id, feature_view, offset=0):
             label_selector=f"job-name={job_id}",
         ).items
         for i, pod in enumerate(pods_list):
-            logger.info(f"Logging output for {feature_view.name} pod {offset+i}")
+            logger.info(f"Logging output for {feature_view.name} pod {offset + i}")
```

Comment:
I think you need to fix the formating of ` + ` and use lazy formatting `%s` for all log messages.

Reply:
By fix, a mean use `# fmt: off` and `# fmt: on`